src/nexmo/__init__.py

```diff
@@ -316,6 +290,13 @@ def get_number_insight(self, params=None, **kwargs):
 
         return self.get(self.api_host, "/number/lookup/json", params or kwargs)
 
+    def get_async_advanced_number_insight(self, params=None, **kwargs):
```

Comment:
I think this change should be in a different PR, it's not related to the signature authentication

Reply:
Done